src/main/scala/mesosphere/marathon/api/v2/GroupsResource.scala

```diff
@@ -250,15 +255,21 @@ class GroupsResource @Inject() (
     (deployment, effectivePath)
   }
 
-  private[this] def authorizedView(root: Group)(implicit identity: Identity): Group = {
+  private[this] def authorizedView(root: Group, embed: java.util.Set[String])(implicit ident: Identity): GroupInfo = {
     def authorizedToViewApp(app: AppDefinition): Boolean = isAuthorized(ViewApp, app)
     val visibleGroups = root.transitiveGroups.filter(group => isAuthorized(ViewGroup, group)).map(_.id)
     val parents = visibleGroups.flatMap(_.allParents) -- visibleGroups
 
-    root.updateGroup { subGroup =>
+    val viewable = root.updateGroup { subGroup =>
       if (visibleGroups.contains(subGroup.id)) Some(subGroup)
       else if (parents.contains(subGroup.id)) Some(subGroup.copy(apps = subGroup.apps.filter(authorizedToViewApp)))
       else None
     }.getOrElse(Group.empty) // fallback, if root is not allowed
+
+    //for backward compatibility, we always render apps and groups, if nothing is given
```

Comment:
Can we maybe get rid of the inline comment and create a constant "DefaultGroupEmbeds" or something like that with the comment?